.github/workflows/prettier.yml

```diff
@@ -0,0 +1,20 @@
+# https://github.com/marketplace/actions/prettier-action
+name: 'Prettier for SEP files'
+
+on:
+  pull_request:
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Check code
+        uses: creyD/prettier_action@v4.3
```

Comment:
Can the @v4.3 be changed to the full git commit sha? See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions for more details on why.

Reply:
Thanks Leigh, good point, I didn't consider that!
Fixed in https://github.com/stellar/stellar-protocol/pull/1469